pkg/reconciler/cluster/apiimporter/apiimporter-controller.go

```diff
@@ -0,0 +1,48 @@
+/*
```

Comment:
(not blocking, naming nit) could these be `apiimporter/controller.go` and `manager.go` (maybe just move `controller-start.go` into `controller.go`? It avoids some stutter, and a somewhat unusual `-` in the filename.

Reply:
See #541 re file name uniqueness. Also +1 to not using dashes in filenames - I commented above about that (should be underscores).